manager/integration/tests/test_basic.py

```diff
@@ -4558,3 +4560,60 @@ def test_backup_failed_disable_auto_cleanup(set_random_backupstore,  # NOQA
                       + volume_name + " is deleted"
     except AssertionError:
         pass
+
+
+@pytest.mark.parametrize(
+    "access_mode,restored_access_mode",
+    [
+        pytest.param("rwx", "rwo"),
+        pytest.param("rwo", "rwx")
+    ],
+)
+def test_backup_volume_restore_with_access_mode(client, access_mode, # NOQA
+                                                restored_access_mode): # NOQA
+    """
+    Test the backup should be save access mode and restore based on the value
+    1. Prepare a healthy volume
+    2. Create a backup for the volume generated in Step 1
+    3. Restore from backup w/o specify the access mode
+       => Validate the access mode should be the same as the Step 1.
+    4. Restore from backup w/ specify the access mode
+       => Validate the access mode should be the same as the specified.
+    """
+    # Step 1
+    test_volume_name = generate_volume_name()
+    client.create_volume(name=test_volume_name,
+                         size=str(DEFAULT_VOLUME_SIZE * Gi),
+                         numberOfReplicas=2,
+                         accessMode=access_mode,
+                         migratable=True if access_mode == "rwx" else False)
```

Comment:
Remove `migratable` argument since this case w/o migratable property support.

https://github.com/longhorn/longhorn-tests/blob/4d01c58adc04fd265fce65b9bcf848e0d7472e69/manager/integration/tests/test_basic.py#L4588-L4591
https://github.com/longhorn/longhorn-tests/blob/4d01c58adc04fd265fce65b9bcf848e0d7472e69/manager/integration/tests/test_basic.py#L4602-L4605
https://github.com/longhorn/longhorn-tests/blob/4d01c58adc04fd265fce65b9bcf848e0d7472e69/manager/integration/tests/test_basic.py#L4602-L4605